DESIGN.md

```diff
@@ -0,0 +1,59 @@
+# Design
+
+This file intends to provide information on how to work with design in the Kiwix android applicaiton. 
+
+## Colors
+
+<img src="https://user-images.githubusercontent.com/22193232/83739760-4ef07080-a656-11ea-8f87-344f48e76324.png" alt="drawing" width="300"/>
+
+The Kiwix colors are black and white with different colored accents. For Kiwix Android, black is accented by different shades of blue:
+
+- ![#000000](https://via.placeholder.com/15/000000/000000?text=+) `#000000`/`black`
+- ![#1565c0](https://via.placeholder.com/15/1565c0/000000?text=+) `#1565c0`/`denim_blue800`
+- ![#42a5f5](https://via.placeholder.com/15/42a5f5/000000?text=+) `#42a5f5`/`denim_blue400`
+- ![#2196F3](https://via.placeholder.com/15/2196F3/000000?text=+) `#2196F3`/`dodger_blue`
+
+> Color indicates which elements are interactive, how they relate to other elements, and their level of prominence. Important elements should stand out the most [1].
+
+For a guide on how to apply colors to UI, see ["Applying color to UI - Material Design"](https://material.io/design/color/applying-color-to-ui.html#backdrop).
+
+## Themes
+Kiwix uses themes to apply styles to views. This means that instead of adding a specific color or style to each view in the application, a theme should be set.
+Themes are defined in `res/values/themes.xml` or `res/values-night/themes.xml` depending on night/day.
+The following steps should be followed to set a color or style of a view.
+1. Define the color or style:
+    * Define the color in `core/colors.xml`. Use good naming (`denim_blue`, `mine_shaft_gray` etc..).
+    * Define the style in `core/styles.xml`. Most styles override theme attributes and should be named to indicate that. This is done by naming the style to match its parent. That is, `Widget.MaterialComponent.*` -> `Widget.KiwixTheme.*`. If multiple styles for the same parent are needed, descriptive names should be used, e.x. `list_item_title`. 
```

Comment:
ah that is not how they are set as the default style, the theme has attributes eg `cardViewStyle` and the style you set there is the default style used for all `CardView`s, we inherit from the parent as they contain sane defaults and we only want to customise it a little bit.

"If multiple styles for the same parent are needed" maybe more: when there are 2 or more of a view that differ stylistically from our default styling but are identical to each visually then it is appropriate to extract a style to keep the XML DRY.

Reply:
Hmm maybe I misread "This is done by naming the style to match its parent.", maybe the intention is better expressed with "For our default styles, applied with theme attributes, the naming convention is `Widget.KiwixTheme.*` and these should ideally inherit from their `Widget.MaterialComponent.*`, should no such parent exist, it is generally advisable to google "how to style [Component]""

Reply:
That sounds better!